src/server/utils/validations.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const moment = require('moment');
+const _ = require('lodash');
+
+
+module.exports = {
+  dateOfBirth: (inputDob) => {
+    let convertedDob = '';
+    if (_.isEmpty(inputDob) || !_.isString(inputDob)) {
+      return convertedDob;
+    }
+    const parsedDob = new Date(inputDob);
+    if (moment().isBefore(parsedDob)) {
+      return convertedDob;
+    }
+    convertedDob = moment(parsedDob).format('MM/DD/YYYY');
+    return convertedDob;
+  },
+  validateString: (data, count, nullable) => {
+    if (nullable && _.isEmpty(data)) {
+      return null;
+    }
+    if (_.isEmpty(data) || !_.isString(data)) {
+      return '';
+    }
+    const validatedString = _.upperFirst(data);
+    if (count && _.isNumber(count)) {
+      return (validatedString.length > count) ? validatedString.slice(0, count) : validatedString;
+    }
+    return validatedString;
+  },
+  validateSsn: (inputSsn) => {
```

Comment:
Can we add some JSDoc comment that summarize the the format validation?  Reading a regexp is really hard.